ipsw_parser/build_identity.py

```diff
@@ -15,17 +15,23 @@
 
 def _extract_dmg(buf: bytes, output: Path) -> None:
     hdiutil = local['hdiutil']
-
+    ipsw = local['ipsw']
```

Comment:
in another part of the code we `try except` the usage of this tool, but i also think we don't need to do it anymore so let's remove that other part